import java.util.Scanner;
 public class IT26102368Lab8Q1A {
    public static void main(String[]args){

    Scanner sc=new Scanner(System.in);
        
     int[] myArray=new int[5];
	 
      System.out.println("Enter 5 numbers :");

           for (int i=0;i<myArray.length; i++){
            
			  System.out.print("Enter Number"+(i+1)+":");
               myArray[i]=sc.nextInt();}

        System.out.println("\nArray in Reverse Order: ");
        
           for (int i =myArray.length -1;i>=0;i--) {
              
			  System.out.print(myArray[i]+ " ");

    
        } 

            System.out.println();
     }
   }